src/helper/ssg/ssg.test.tsx

```diff
@@ -367,7 +367,7 @@ describe('saveContentToFile function', () => {
   const gzFileArrayBuffer = gzFileBuffer.buffer.slice(
     gzFileBuffer.byteOffset,
     gzFileBuffer.byteLength + gzFileBuffer.byteOffset
-  )
+  ) as ArrayBuffer
```

Comment:
The `as ArrayBuffer` is required because `Buffer.buffer` returns `ArrayBufferLike` in TypeScript 5.9, but we know it's always `ArrayBuffer` in Node.js context.